src/cache/cache_las.c

```diff
@@ -562,7 +573,7 @@ __las_squash_modifies(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_UPDATE **u
     }
     cursor->set_value(
       cursor, start_upd->durable_ts, start_upd->prepare_state, WT_UPDATE_STANDARD, &las_value);
-    WT_ERR(cursor->insert(cursor));
+    WT_ERR(cursor->update(cursor));
```

Comment:
Please add a comment, as to why we need to use "update" instead of "insert".

Reply:
As the original las insert was done with update to let the page to pinned. I followed the same for the newly added code for squash modifies. I added a comment explaining the same in the code.